src/features/perfBrowser/usePerfBrowser.ts

```diff
@@ -262,6 +262,12 @@ const usePerfBrowser = (
         (v) => v
     );
 
+    useEffect(() => {
+        if (seriesGroupsWithRounds.length > 0) {
+            setXAxisMode('round');
+        }
+    }, [seriesGroupsWithRounds.length, setXAxisMode]);
+
```

Comment:
If I understand correctly, we load the possibilities (with `useSyncedState<XAxisModeT>`) and then this `useEffect` selects "round" as the default.

Would there be some more elegant way?

Reply:
Not exactly, `useSyncedState` syncs the current value selected by using the value in the url, sets the default at "rank" if there's no value selected and syncs the url with the new value when changing value with the select menu.
This `useEffect` checks if there's series with rounds to change the value which defaulted to "rank" to "round".